lighthouse-core/gather/gatherers/image-usage.js

```diff
@@ -36,9 +36,46 @@ function collectImageElementInfo() {
       clientHeight: element.clientHeight,
       naturalWidth: element.naturalWidth,
       naturalHeight: element.naturalHeight,
+      isCss: false,
       isPicture: element.parentElement.tagName === 'PICTURE',
     };
   });
+
+  // Chrome normalizes background image style from getComputedStyle to be an absolute URL in quotes.
+  // Only match basic background-image: url("http://host/image.jpeg") declarations
+  const CSS_URL_REGEX = /^url\("([^"]+)"\)$/;
+  // Only find images that aren't specifically scaled
+  const CSS_SIZE_REGEX = /(auto|contain|cover)/;
+  const cssImages = [...document.querySelectorAll('*')].reduce((images, element) => {
```

Comment:
we should probably make this shadow dom-aware: `document.querySelectorAll('html /deep/ *')`.

https://github.com/GoogleChrome/lighthouse/blob/b06e808a5eb6b0c077bd0d855b9ff469aad5b056/lighthouse-core/gather/gatherers/dobetterweb/domstats.js#L111